utils/common.py

```diff
@@ -138,25 +138,43 @@ class country_us(object):
         }
 
     st_abbr = {
-        'DR': 'DRIVE',
-        'RD': 'ROAD',
-        'BLVD': 'BOULEVARD',
-        'ST': 'STREET',
-        'STE': 'SUITE',
-        'APTS': 'APARTMENTS',
         'APT': 'APARTMENT',
-        'CT': 'COURT',
-        'LN': 'LANE',
+        'APTS': 'APARTMENTS',
         'AVE': 'AVENUE',
+        'BLVD': 'BOULEVARD',
+        'BR': 'BRIDGE',
         'CIR': 'CIRCLE',
-        'PKWY': 'PARKWAY',
+        'CT': 'COURT',
+        'DR': 'DRIVE',
         'HWY': 'HIGHWAY',
-        'SQ': 'SQUARE',
-        'BR': 'BRIDGE',
+        'HW': 'HIGHWAY',
         'LK': 'LAKE',
+        'LN': 'LANE',
+        'RD': 'ROAD',
         'MT': 'MOUNT',
         'MTN': 'MOUNTAIN',
+        'PKWY': 'PARKWAY',
         'PL': 'PLACE',
         'RTE': 'ROUTE',
+        'SQ': 'SQUARE',
+        'ST': 'STREET',
+        'STE': 'SUITE',
+        'TPKE': 'TURNPIKE',
         'TR': 'TRAIL'
         }
+
+    def _lookup_words(input_text):
```

Comment:
I feel like I'm having something fail to strip and/or replace here... or just before here.

Reply:
There is no formatting here for punctuation, but considering that the unexpected text made it out in upper-case ... that would indicate it did come through this function.  It may be that the punctuation failed to be removed and this code cannot ID it as being a key in the list, thus triggering the except clause.

Reply:
Its possible that its not triggering the except, but rather not triggering the IF statement.

Reply:
Looks like what ever failure is happening, its before this function as values both trigger the if and the else (added locally for debug) with improper conversion.